lib/boombox/bin.ex

```diff
@@ -56,7 +56,13 @@ defmodule Boombox.Bin do
   """
   @type input() ::
           (path_or_uri :: String.t())
-          | {:mp4, location :: String.t(), transport: :file | :http}
+          | {:mp4 | :aac | :wav | :mp3 | :ivf | :ogg | :h264 | :h265, location :: String.t()}
```

Comment:
As discussed - it would be good not to repeat these types definitions here and in `lib/boombox.ex` but we can leave it for now